File: QRStockMateApp/app/src/main/java/com/example/qrstockmateapp/api/models/message.kt
package com.example.qrstockmateapp.api.models

data class Message(
    val id: Int,
    val code: String,
    val senderContactId: Int,
    val receiverContactId: Int,
    val content: String,
    val sentDate: String, // O puedes usar DateTime si estás utilizando alguna biblioteca de fechas en Kotlin
    val type: Int
)

File: QRStockMateWeb/src/app/interfaces/message.ts
export interface Message {
    id: number;
    code: string;
    senderContactId: number;
    receiverContactId: number;
    content: string;
    sentDate: Date;
    type: TypeFile;
  }
  export enum TypeFile {
    Text,
    Audio,
    File,
    Image
  }
  